README.md

```diff
@@ -47,6 +47,13 @@ $ cd my_project/
 $ symfony serve
 ```
 
+If you have cloned the [GitHub repository](https://github.com/symfony/demo), you
+must install the Composer dependencies.
+
+```bash
+$ composer install
```

Comment:
What about using Symfony CLI in the command? It will use the correct PHP version
```suggestion
$ symfony composer install
```

Reply:
Yes, seems better in this case.

Reply:
I'd prefer to not mix `symfony` binary with Composer ... because people who will use the Composer installation method is people who don't want or can't download the Symfony CLI, so they won't have access to the `symfony` binary. Thanks.